content/en/docs/How Tos/metrics.md

```diff
@@ -6,14 +6,18 @@ description: >
   How to export metrics from Baker
 ---
 
-During its execution, a Baker pipeline exports metrics about the Go runtime as
-well as general metrics giving an high-level overview of the ongoing job.
-
-More specific metrics are also exported on a per-component basis. To that effect, 
-`baker.Input`, `baker.Filter`, `baker.Output` and `baker.Upload` all have a `Stats` 
-method. `Stats` is called every second and the component is expected to return both
-a predefined set of metrics and a [`baker.MetricsBag`](https://pkg.go.dev/github.com/AdRoll/baker#MetricsBag)
-containing metrics of arbitrary name and types.
+During its execution, Baker can publish various kind of metrics that may be used
+to monitor a pipeline in execution. The metrics exported range from numbers giving 
```

Comment:
`execution` is repeated twice, I'd probably remove `During its execution` at the beginning